backend/src/app.js

```diff
@@ -1,6 +1,7 @@
 const express = require("express");
 const path = require("path");
 const cors = require("cors");
+// eslint-disable-next-line import/no-unresolved
```

Comment:
eslint rules should not be disabled. did you run `npm run setup` from the root folder ?